cmd/cli/cmd/manifest/generate/interfaces/interface.go

```diff
@@ -3,9 +3,10 @@ package interfaces
 import (
```

Comment:
Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 29